fpo-api/fpo_api/urls.py

```diff
@@ -20,5 +20,6 @@
     path("health/", views.health)
 ]
 
-if (os.getenv("FRONT_END", "prod") == "dev"): 
+if (os.getenv("FRONT_END", "prod") == "dev"):
+    urlpatterns.append(path("family-law-act/family-law-act/api/v1/", include("api.urls"))) 
```

Comment:
Is the duplicated path required for `dev`?

Reply:
Yes,
For dev environment: I run the API and DB in docker, I run the front-end in dev on port 8084, 
I have to use this path (duplicate:  'family-law-act/family-law-act/api/v1/' ) in order to hit the API.